stories/core/LabeledInput.stories.tsx

```diff
@@ -146,3 +147,29 @@ Inline.args = {
   status: 'negative',
   displayStyle: 'inline',
 };
+
+export const HybridLayout: Story<LabeledInputProps> = ({
+  svgIcon,
+  ...rest
+}) => {
+  const ref = React.useRef(null);
+  return (
+    <>
+      <LabeledInput
+        placeholder='Enter text here'
+        label='This is a label'
+        displayStyle='inline'
+        svgIcon={<div ref={ref}>{svgIcon}</div>}
+        message='Block layout with inline icon'
+        {...rest}
+      />
+      <Tooltip reference={ref} content='Placeholder' />
```

Comment:
Does it work passing `Tooltip` with icon as a children to `svgIcon`?

Reply:
no, because svgIcon component does not forward ref.

Reply:
So have it wrapped with `div` like it is now.

Reply:
It does not work like that...